locales-src/asrouter.ftl

```diff
@@ -90,17 +90,41 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
+
+cfr-whatsnew-tracking-protect-title = Protect yourself from trackers
+cfr-whatsnew-tracking-protect-body = { -brand-short-name } blocks many common social and cross-site trackers that follow what you do online.
+cfr-whatsnew-tracking-protect-link-text = View your protection report
+
 # This string is displayed before a large numeral that indicates the total number
 # of tracking elements blocked. Don’t add $blockedCount to
 # your localization, because it would result in the number showing twice.
-cfr-whatsnew-tracking-protections-title =
+cfr-whatsnew-tracking-blocked-title =
   { $blockedCount ->
     [one] Tracker blocked
    *[other] Trackers blocked
   }
-cfr-whatsnew-tracking-protections-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-link-text = View Protection Report
+
+cfr-whatsnew-lockwise-backup-title = Back up your passwords
+cfr-whatsnew-lockwise-backup-body = Now generate secure passwords you can access anywhere you sign in.
+cfr-whatsnew-lockwise-backup-link-text = Turn on backups
+
+cfr-whatsnew-lockwise-take-title = Take your passwords with you
+cfr-whatsnew-lockwise-take-body = The { -lockwise-brand-short-name } mobile app lets you securely access your backed up passwords from anywhere.
+cfr-whatsnew-lockwise-take-link-text = Get the app
 
 cfr-whatsnew-panel-header = What’s New
+cfr-whatsnew-dismiss-button = Dismiss
+
+# Variables:
+#  $version (String) - Firefox browser version
+cfr-whatsnew-firefox-notes-title = { -brand-short-name } Version { $version }
+cfr-whatsnew-firefox-notes-link-text = Read the release notes
```

Comment:
Not all locales have release notes… I suppose we should make sure to remotely target this panel message to the appropriate locales. @piatra can we optionally not have a link and just have the body as a release-notes 1-liner?

Reply:
Oh, I saw flod's comment on my previous line that I removed of only en-US has release notes. So I suppose this is another candidate for maybe not exposing to all locales and just do it from the remote en-US targeted message.

Reply:
That's a product decision, it could be still worth sending people to release notes, even if they're not translated.

Reply:
If it makes it easier: I see that string was removed from the designs and we were not planning on implementing it either. We can defer for later?

Reply:
Moved to the plain message "hero"

Reply:
Jim says we'll keep this generic "Read the release notes" link to unlocalized and see how people interact which could be a signal in how we do release notes going forwards.